geode-core/src/main/java/org/apache/geode/cache/query/security/MethodInvocationAuthorizer.java

```diff
@@ -58,4 +59,6 @@
    *         {@code false} otherwise.
    */
   boolean authorize(Method method, Object target);
+
+  void initialize(Cache cache, Set<String> parameters);
```

Comment:
Can you add `default` to the `initialize` method so actual implementations of the interface are not forced to manually implement the method if they don't need to?. Similar to what we do for the [Function](https://github.com/apache/geode/blob/rel/v1.10.0/geode-core/src/main/java/org/apache/geode/cache/execute/Function.java) interface.

Reply:
Done!